.github/workflows/plugins-test.yaml

```diff
@@ -118,6 +118,8 @@ jobs:
         run: bash test/plugin/run.sh netty-socketio-scenario
       - name: Run rabbitmq-scenario (12)
         run: bash test/plugin/run.sh rabbitmq-scenario
+      - name: Run gson-scenario (6)
+        run: bash test/plugin/run.sh gson-scenario
```

Comment:
You add this to the group, but you haven't change the group name.

Reply:
You  mean i change the name to 'Webflux_SocketIO_GSON', am i right ?

Reply:
Yes, this name. But should be `Toolkit_Webflux_SocketIO_RabbitMQ_GSON`

Reply:
OK, Got it.